samples/meetings-app-icon-badging/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meetings-app-icon-badging-csharp",
+    "source": "officeDev",
+    "title": "Meetings-App-Icon-Badging",
+    "shortDescription": "Illustrates how to implement app icon badging \"in-meeting notification\" using \"targeted meeting notification\" API",
```

Comment:
Illustrates how to => This sample shows how to..............

Reply:
fixed